openshift/catalogd/kustomize/overlays/openshift/olmv1-ns/patches/manager_deployment_certs.yaml

```diff
@@ -24,4 +18,4 @@
   value: "--tls-key=/var/certs/tls.key"
 - op: add
   path: /spec/template/spec/containers/0/args/-
-  value: "--ca-certs-dir=/var/trusted-cas"
```

Comment:
Why are we changing the name here?

Reply:
Avoid confusion since trusted-ca-bundle and service-ca are combined?

Reply:
But I don't have a strong opinion here, especially if we're going to end up splitting these up and using separate flags for various CA usages.

Reply:
But they were combined before. The latest solution I have avoids the use of the flags...